iidm/iidm-xml-converter/src/main/java/com/powsybl/iidm/xml/XMLExporter.java

```diff
@@ -88,7 +99,8 @@ public void export(Network network, Properties parameters, DataSource dataSource
                     .setForceBusBranchTopo("true".equals(parameters.getProperty(FORCE_BUS_BRANCH_TOPO_PROPERTY, "false")))
```

Comment:
I asked you to log a warning if the parameter is present in the configuration file